hocr-extract-images

```diff
@@ -13,7 +13,7 @@ def assoc(key,list):
 
 def get_text(node):
     textnodes = node.xpath('.//text()')
-    s = string.join([text for text in textnodes])
+    s = ''.join([text for text in textnodes])
```

Comment:
True, but it's a trivial change, juggling the commits to get this patch into the commit before is not. Can do if you think it's important.